src/api/spec/bootstrap/features/webui/groups_spec.rb

```diff
@@ -2,47 +2,74 @@
-    # TODO: Enable after migration is finished
-    skip_if_bootstrap
+  let(:user_1) { create(:confirmed_user, login: 'eisendieter') }
+  let!(:group_1) { create(:group, title: 'test_group', users: [admin, user_1]) }
+  let!(:group_2) { create(:group, title: 'test_group_b') }
 
+  before do
     login admin
+  end
+
+  def group_in_datatable(page, group)
+    expect(page).to have_link(group.title, href: group_show_path(group))
+    group.users.each { |user| expect(page).to have_link(user.login, href: user_show_path(user)) }
+    expect(page).to have_link('Edit', href: group_edit_title_path(group))
+  end
+
+  scenario 'visit groups index page' do
     visit groups_path
 
+    [group_1, group_2].each do |group|
+      group_in_datatable(page, group)
+    end
+
     expect(page).to have_content('Showing 1 to 2 of 2 entries')
-    click_link('test_group_b')
+  end
+
+  scenario 'visit group show page' do
+    visit group_show_path(group_1)
+
     expect(page).to have_content('Incoming Reviews')
+    expect(page).to have_content('Incoming Requests')
+    expect(page).to have_content('All Requests')
     find('#group-members-tab').click
-    expect(page).to have_content('This group does not contain users.')
-    expect(page).to have_link('Add User')
 
-    visit groups_path
-    expect(page).to have_content('Showing 1 to 2 of 2 entries')
-    within :xpath, "//tr[@id='group-test_group']" do
-      click_link(admin.login)
-    end
-    expect(page).to have_current_path(user_show_path(admin))
+    expect(page).to have_link('Edit Group', href: group_edit_title_path(group_1))
+    group_1.users.each { |user| expect(page).to have_link(user.login, href: user_show_path(user)) }
   end
 
-  scenario 'add a user' do
-    # TODO: Enable after migration is finished
-    skip_if_bootstrap
+  scenario 'create a group' do
+    visit groups_path
 
-    login admin
+    click_link('Create Group', href: group_new_path)
+
+    new_group_title = 'group_123'
+    fill_in('group_title', with: new_group_title)
+    # Typing a comma after a user login selects it (just like clicking on the autocomplete menu popping up). It's a built-in feature of the tokenfield
+    fill_in('group_members_tag', with: "#{admin},#{user_1},")
+
+    expect { click_button('Create') }.to change(Group, :count).by(1)
+    expect(page).to have_content("Group '#{new_group_title}' successfully created.")
+    group_in_datatable(page, Group.find_by(title: new_group_title))
+  end
+
+  scenario 'edit a group' do
     visit groups_path
 
-    within :xpath, "//tr[@id='group-test_group_b']" do
-      click_link 'test_group_b'
-    end
+    click_link('Edit', href: group_edit_title_path(group_1))
 
-    find('#group-members-tab').click
-    click_link 'Add User'
-    fill_in 'group_userid', with: 'eisendieter'
-    click_button('Accept')
-    expect(page).to have_content('eisendieter')
+    # Remove all users from tokenfield
+    2.times { first('button.tag-remove').click }
```

Comment:
IIRC, you could use `find_all` instead

Reply:
`find` does find all elements, but clicking them doesn't work because it says something like `Ambiguous match, found 2 elements matching (...)`.

Reply:
[all / find_all](https://www.rubydoc.info/github/jnicklas/capybara/Capybara/Node/Finders#all-instance_method) yields the elements. So you probably could write `
all('button.tag-remove').each { |element| element.click }` (*untested*)

Reply:
This worked, thank you!